helpers/base.sh

```diff
@@ -452,5 +452,6 @@ do_exit() {
 # Returns:
 #     None
 set_traps() {
-    trap cleanup 1 2 3 6 ERR
+    trap "cleanup && exit 0" 2
```

Comment:
The exit status code of the interrupted by Ctrl-C process shouldn't be `0`. See [this discussion](https://unix.stackexchange.com/questions/251996/why-does-bash-set-exit-status-to-non-zero-on-ctrl-c-or-ctrl-z), for example.